interface/src/mvd-hosting.d.ts

```diff
@@ -89,17 +89,18 @@ declare namespace MVDHosting {
     performLogin(username: string, password: string): Observable<Response>;
   }
 
-  export interface NotificationManagerInterface {
-    push(notification: Notification): void;
-    pop(): Notification | void;
-    getAll(): Notification[] | void;
-    getAllByCategory(type: MVDHosting.NotificationType): Notification[] | void;
+  export interface ZoweNotificationManagerInterface {
+    push(notification: ZoweNotification): void;
```

Comment:
push=notify
updateHandlers=make this _updateHandlers, an internal function that is not part of the interface since you seem to mean to not want people to use this directly.
removeFromCache=dismissNotication